graphjet-core/src/main/java/com/twitter/graphjet/datastructures/Pair.java

```diff
@@ -1,4 +1,4 @@
-package com.twitter.graphjet.algorithms;
+package com.twitter.graphjet.datastructures;
```

Comment:
Consider just using one of the many publicly available Pair implementations: https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/tuple/Pair.html

I've also often found that there is many times a better solution than using a pair --- I imagine you'll use this to return (node, metadata), in which case it may be better to either create a custom class to avoid auto-boxing, or to figure out an alternate API.